.gitignore

```diff
@@ -1,128 +1,13 @@
-### Project specific files
-bin/
-derby_db/
-access_logs/
-logs/
-release/
-apidoc/
-classes/
-temp/
-temp-build/
-testclasses/
-test-results/
-savedGames/
-
-## Derby
-derby.log
-ta_users/
-
-### JetBrains template
-# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion
-
-*.iml
-
-## Directory-based project format:
-.idea/
-# if you remove the above rule, at least ignore the following:
-
-# User-specific stuff:
-# .idea/workspace.xml
-# .idea/tasks.xml
-# .idea/dictionaries
-
-# Sensitive or high-churn files:
-# .idea/dataSources.ids
-# .idea/dataSources.xml
-# .idea/sqlDataSources.xml
-# .idea/dynamic.xml
-# .idea/uiDesigner.xml
-
-# Gradle:
-# .idea/gradle.xml
-# .idea/libraries
-
-# Mongo Explorer plugin:
-# .idea/mongoSettings.xml
-
-## File-based project format:
-*.ipr
-*.iws
-
-## Plugin-specific files:
-
-# IntelliJ
-/out/
-
-### Eclipse template
-*.pydevproject
-.metadata
+.classpath
 .gradle
-bin/
-tmp/
-*.tmp
-*.bak
-*.swp
-*~.nib
-local.properties
-.settings/
-.loadpath
-temp_eclipse/
-classes.jar
-
-# Vi tmp files
-*~
-
-# Eclipse Core
+.idea
 .project
+.settings
 
-# External tool builders
-.externalToolBuilders/
-
-# CDT-specific
-.cproject
-
-# JDT-specific (Eclipse Java Development Tools)
-.classpath
-
-# Java annotation processor (APT)
-.factorypath
-
-# PDT-specific
-.buildpath
-
-# Checkstyle plugin
-.checkstyle
```

Comment:
This should still be included.  It's generated by the Checkstyle Eclipse plugin.